src/server/src/modules/job.ts

```diff
@@ -481,3 +480,99 @@ export const getApplicants = async (
 
     return getApplicantsResult;
 };
+
+/**
+ * @description Returns at most 20 newest jobs that match with student's department
+ *              and class standing.
+ * @param {number} studentId - id of student
+ * @param {number} page - page index
+ * @param {number} numOfItems - number of items per page
```

Comment:
This function doesn't receive a numOfItems parameter

Reply:
Done.